wsgi/odyssy/odyssy/settings.py

```diff
@@ -36,6 +36,7 @@
 DEBUG = not ON_OPENSHIFT
 
 ALLOWED_HOSTS = [
+    '127.0.0.1',
```

Comment:
@Prakash2403: Make it as `*` when `DEBUG` is `True` and let it be as it is otherwise.